stylelint-polaris/README.md

```diff
@@ -523,64 +523,117 @@ Disallows use of legacy mixin map data.
 
 #### z-index/declaration-property-value-allowed-list
 
-```diff
+> Even though there might not be a z-index equivalent in [Polaris tokens](https://polaris.shopify.com/tokens/z-index), try and use the Polaris depth scale before creating your own custom one.
 
+```diff
+// Don't
+- z-index: 1;
+// Do
++ z-index: var(--p-z-1);
 ```
 
 #### z-index/function-disallowed-list
 
-```diff
+> Even though there might not be a z-index equivalent in [Polaris tokens](https://polaris.shopify.com/tokens/z-index), try and use the Polaris depth scale before creating your own custom one.
 
+```diff
+// Don't
+- z-index: z-index(content);
+// Do
++ z-index: var(--p-z-1);
 ```
 
 #### z-index/global-disallowed-list
 
-```diff
+> Even though there might not be a z-index equivalent in [Polaris tokens](https://polaris.shopify.com/tokens/z-index), try and use the Polaris depth scale before creating your own custom one.
 
+```diff
+// Don't
+- z-index(toast, $fixed-element-stacking-order);
+// Do
++ z-index: var(--p-z-1);
 ```
 
 ### Layout
 
 #### layout/declaration-property-value-disallowed-list
 
-```diff
+> There are many ways to use [Polaris components](https://polaris.shopify.com/components) to compose desired layouts. Please explore the layout components before writing custom styles.
 
+```diff
+// Don't
+- width: 100%;
+// Do
++ <AlphaStack fullWidth />
 ```
 
 #### layout/function-disallowed-list
 
 ```diff
-
+// Don't
+- height: top-bar-height();
+// Do
++ height: 56px;
 ```
 
 #### layout/at-rule-disallowed-list
 
 ```diff
-
+// Don't
+- @include print-hidden;
+// Do
++ @media print {
++   display: none;
++ }
 ```
 
 #### layout/property-disallowed-list
 
+> There are many ways to use [Polaris components](https://polaris.shopify.com/components) to compose desired layouts. Please explore the layout components before writing custom styles.
+
 ```diff
+// Don't
+- display: grid;
+// Do
++ <Grid />
```

Comment:
even though these lines of code wouldn't be close to each other and the `<Grid />` is clearly a stub, is it ok to show it like this? I think it gets across the message still but open to other opinions

Reply:
I think recommending the component equivalent makes the most sense here 💯 The `Grid` is going to be deprecated as the Layout project team decided the `Columns` component does everything `Grid` can and they don't want the confusion between which one to use.

```suggestion
+ <Columns {...props} />
```